tests/basic/print_cpuinfo/print_cpuinfo.c

```diff
@@ -0,0 +1,58 @@
+#define _GNU_SOURCE
+
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+#include <errno.h>
+#include <stdlib.h>
+
+int test(void)
+{
+    FILE *fp = fopen("/proc/cpuinfo", "r");
+    if (fp == NULL) {
+        printf("Unable to open /proc/cpuinfo\n");
+        return -1;
+    }
+
+    size_t n = 0;
+    char *line = NULL;
+
+    int ret = 0;
+
+    while (getline(&line, &n, fp) > 0) {
+        printf("%s", line);
+
+        if (strstr(line, "cpu cores")) {
+            unsigned int num_cores = 0;
+            if (sscanf(line, "cpu cores       : %u", &num_cores) != 1) {
+                printf("Failed to sscanf cpu cores information\n");
+                ret = -1;
+                break;
+            }
+            if (num_cores != 1) {
```

Comment:
If we run with more than 1 ethread, this test will fail?  Please pass `SGXLKL_ETHREADS` in as an environment variable that's propagated to the program and check that the value matches.

Reply:
Done. It is passed as an argument because passing it as an environment variable was a bit tricky and I couldn't get it working easily. Maybe I just missed something.